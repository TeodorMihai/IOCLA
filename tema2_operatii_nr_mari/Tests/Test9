CAFEBABE123434567344556722FABDEFFF1353ADADADADADADADADADADADADADACDCDCDCDCDCDCDCDCDCDCDCDCDCDCDCDCDCDCDCDCDCDCDCDCDBBBBBBBB345456567678678452343454565676787897894 >> 1F
