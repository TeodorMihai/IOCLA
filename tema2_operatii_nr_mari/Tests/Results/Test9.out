94788987676765454543234578866767654545B3BBBBBBDBDCDCDCDCDCDCDCDCDCDCDCDCDCDCDCDCDCDCDCDCDCDCDCDCACADADADADADADADADADADADADAD5313FFEFBDFA226755447356343412BEBAFECA
1F
CFCECA8A8A86468AF00CCFCECA8A8A66777777B7B9B9B9B9B9B9B9B9B9B9B9B9B9B9B9B9B9B9B9B9B9B9B9B9595B5B5B5B5B5B5B5B5B5B5B5B5BA726FEDF7BF545CEAA88E6AC6868247C75FD9501
195FD757C246868ACE688AACE45F57BDFFE26A75B5B5B5B5B5B5B5B5B5B5B5B5B59B9B9B9B9B9B9B9B9B9B9B9B9B9B9B9B9B9B9B9B9B9B9B9B9B7777777668A8ACACECF0CF08A46868A8ACACECF
